CMakeLists.txt

```diff
@@ -94,9 +94,22 @@ target_compile_definitions(${TARGET_NAME}
         JUCE_USE_CURL=0
         JUCE_APPLICATION_NAME_STRING="$<TARGET_PROPERTY:${PROJECT_NAME},JUCE_PRODUCT_NAME>"
         JUCE_APPLICATION_VERSION_STRING="$<TARGET_PROPERTY:${PROJECT_NAME},JUCE_VERSION>"
+)
+
+if(MSVC) 
```

Comment:
There's no need to have a special case for MSVC, can just remove the `-D` from all of them and it will work as expected. It should never have been there in the first place, it just happens that it doesn't break things when building with `clang` or `gcc`.

Reply:
I've quickly fixed this now.